presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -737,8 +754,8 @@ private ConnectorTableMetadata doGetTableMetadata(ConnectorSession session, Sche
                     throw e;
                 }
             }
-        }
-        return columns.build();
+            return null;
+        }).filter(Objects::nonNull);
```

Comment:
Per general style guidelines, the `.filter` and `.map` above should be preceded with a new line.
I'd recommend to do this as a follow up though, not to incur bigger diff within this PR.